lib/graphql/card.ts

```diff
@@ -0,0 +1,101 @@
+import { Card as CardModel, Query } from "./schema";
+import { Model } from "./model";
+import { ResultPage } from "./resultPage";
+import { KontistSDKError } from "../errors";
+import { CardOptions } from "./types"
+
+
+const CARD_FIELDS = `
```

Comment:
should we expose all of these fields in SDK? I don't know much about the business logic behind these so I just exposed everything (cc @p-janik 😛)

Reply:
Not all of them have to be exposed,  will double-check those values 👍

Reply:
Have removed the ones we agreed upon in this commit: 08bd85269213474117035208c67d728dbb60d883